vspec/model/constants.py

```diff
@@ -59,6 +66,14 @@ class StringStyle(Enum, metaclass=EnumMetaWithReverseLookup):
 
 
 class Unit(Enum, metaclass=EnumMetaWithReverseLookup):
+    """Data Unit Types
```

Comment:
I would rebase and split these doc strings into a separate commit.  It seems unrelated to the other code changes.

Reply:
or maybe... why not put it together with the previous change, where you did a significant rewrite of the Unit constants.  Feels like it more belongs there.

Reply:
let me know if they should be merged or not

Reply:
> let me know if they should be merged or not

In my opinion, no need to squash them together (so I didn't ask for it).